app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CustomersController < ApplicationController
+
+  def show
+    @customer = Customer.find(params[:id])
+    @items = @customer.items
```

Comment:
You created this ivar, but then in the form, you didn't reference it, you called `@customer.items`